### DISCUSSION TITLE
I have a question about the websites associated with this GitHub project.

### DISCUSSION THREAD

**Discussion by GG-GhostGaming** [tteck/Proxmox#3855] [Q&A] [answered]
Do you know of this website and is it associated with this project because it is actively using this project's branding, icons, references and links?

I want to know because I was using it interchangeably with the website listed on this project to set up my CTs/VMs  and I've (stupidly) been under the impression that it was associated with this project and it was just an overhaul of the website in progress.

Edit: Removed link

**Comment by tteck** [comment, marked as answer]
That site is not affiliated with our project and appears to be exploiting our popularity for attention or, worse, to potentially distribute malicious code. For accurate and safe information, always refer to our official resources at https://helper-scripts.com/ or https://tteck.github.io/Proxmox/.
